sites/main-site/src/content/blog/2025/modules-ontology.mdx

```diff
@@ -0,0 +1,323 @@
+                    pattern: "true or false"
+        output:
+            - bam:
+                - meta:
+                    type: file
+                    description: Groovy Map containing sample information
+                - "*.bam":
+                    type: file
+                    description: Output BAM file containing read alignments
+                    pattern: "*.{bam}"
+            - cram:
+                - meta:
+                    type: file
+                    description: Groovy Map containing sample information
+                - "*.cram":
+                    type: file
+                    description: Output CRAM file containing read alignments
+                    pattern: "*.{cram}"
+            - csi:
+                - meta:
+                    type: file
+                    description: Groovy Map containing sample information
+                - "*.csi":
+                    type: file
+                    description: Optional index file for BAM file
+                    pattern: "*.{csi}"
+            - crai:
+                - meta:
+                    type: file
+                    description: Groovy Map containing sample information
+                - "*.crai":
+                    type: file
+                    description: Optional index file for CRAM file
+                    pattern: "*.{crai}"
+            - versions:
+                - versions.yml:
+                    type: file
+                    description: File containing software versions
+                    pattern: "versions.yml"
+        ...
+        ```
+    </div>
+
+</div>
+
+We also introduced linting checks to nf-core/tools to ensure the proper structure of `meta.yml` files.
+
+Together with this linting checks, we also introduced a new flag to the `nf-core modules lint` command, `--fix`.
+This flag will try to fix all the possible lint failures related to the `meta.yml` file by updating it.
+
+### Introducing ontologies
+
+Together with these changes, we also added the [`bio.tools`](https://bio.tools/) identifier of the tool.
+
+```yml title="meta.yml" {13}
+name: bwa_mem
+
+    ...
+
+tools:
+    - bwa:
+          description: |
+              BWA is a software package for mapping DNA
+              sequences against a large reference genome, such as the human genome. homepage: http://bio-bwa.sourceforge.net/
+          documentation: https://bio-bwa.sourceforge.net/bwa.shtml
+          arxiv: arXiv:1303.3997
+          licence: ["GPL-3.0-or-later"]
+          identifier: "biotools:bwa"
+```
+
+This bio.tools ID opened new possibilities, such as being able to grab the inputs and outputs of that tool and find the ontology terms for input and output files.
+
+We now use this information to generate a suggestion of input and output channels when creating a module with `nf-core modules create`.
+(available as of February 2025 on the `dev` version of nf-core/tools)
```

Comment:
```suggestion
We now use this information to generate a suggestion of input and output channels when creating a module with `nf-core modules create`
(available as of February 2025 on the `dev` version of nf-core/tools and it will be released as part of nf-core/tools 3.3.0)
```